tests/platforms/kubernetes/app_description.go

```diff
@@ -25,4 +25,5 @@ type AppDescription struct {
 	DaprCPURequest    string
 	DaprMemoryLimit   string
 	DaprMemoryRequest string
+	Namespace         *string
```

Comment:
why * here?

Reply:
Spoke in person, but for record keeping's sake it's to allow a nil comparison in the code. The default namespace is what is used by all the e2e tests except for these few here. Instead of changing all tests to provide a namespace, this allows us to omit it and define a default in 1 place. In Go, you can't compare a string and nil, it has to be *string.